AttitudeManager/Inc/AM.hpp

```diff
@@ -12,19 +12,32 @@
 #include "AM_ControlAlgorithm.hpp"
 #include "CommonDataTypes.hpp"
 
+#include <vector>
+
 namespace AM {
 
 class AttitudeManager {
    public:
-    AttitudeManager(Flightmode* control_algorithm) : control_algorithm(control_algorithm){};
+    AttitudeManager(Flightmode* control_algorithm);
 
     void runControlLoopIteration(const AttitudeManagerInput& instructions);
 
    private:
     AttitudeManager();
+    void outputToMotor(config::ControlAxis_t axis, uint8_t percent);
+
+    Flightmode* control_algorithm_;
 
-    Flightmode* control_algorithm;
+    std::vector<MotorInstance_t> motorInstances_[4]; // 4 is the size of config::ControlAxis_t enum
```

Comment:
You should probably zero-initialize this.

I'm also not convinced that a vector is required here... more below